### PR TITLE
TW-991: creating DM with no encryption by default, hiding create chat on web

### PR DESCRIPTION
**Pull request by imGok**
#991

- Disable Encryption by default
- Hide new group chat on web only
- Fix new group chat redirection with mixin

# WEB

## Unencrypted DM

https://github.com/linagora/twake-on-matrix/assets/48354990/f5797c43-7fd6-47ba-bc80-ad447776bd23

## New Group Chat

https://github.com/linagora/twake-on-matrix/assets/48354990/9c859293-f076-4845-b72a-c3d470e54628

# MOBILE

image

### CODE REVIEW

File: lib/pages/new_private_chat/widget/expansion_list.dart [hunk 1]
```diff
@@ -177,9 +169,7 @@ class ExpansionList extends StatelessWidget {
       child: Column(
         crossAxisAlignment: CrossAxisAlignment.start,
         children: [
-          _NewGroupButton(
-            onPressed: goToNewGroupChat,
-          ),
+          ..._buildResponsiveButtons(context),
           _GetHelpButton(),
```
Comment (hoangdat) [inline lib/pages/new_private_chat/widget/expansion_list.dart:173]: IMO, please remove it, no function for it now

Comment (imGok) [reply lib/pages/new_private_chat/widget/expansion_list.dart:173]: button doesn't appear on web so IMO it's cleaner than writing this 6 times
```dart
    if (getIt.get<ResponsiveUtils>().isSingleColumnLayout(context)) ;
      _NewGroupButton(
        onPressed: goToNewGroupChat,
      ),
```

Comment (hoangdat) [reply lib/pages/new_private_chat/widget/expansion_list.dart:173]: no, I mean `_GetHelpButton(),`

**Comment by sherlockvn** [comment]
i think you forgot to disable encryption option in new private chat

**Review by sherlockvn — APPROVED** [review]

**Pull request merged**